pyproject.toml

```diff
@@ -45,20 +49,77 @@ plotting = [
     "imageio-ffmpeg",
 ]
 
-[tool.flit.module]
-name = "ansys.dpf.core"
-
 [project.urls]
 Homepage = "https://dpf.docs.pyansys.com/"
 Documentation = "https://dpf.docs.pyansys.com/"
 Source = "https://github.com/ansys/pydpf-core"
 Tracker = "https://github.com/ansys/pydpf-core/issues"
 
+[tool.setuptools]
+package-dir = {"" = "src"}
+packages = [
```

Comment:
@PProfizi I think you could add a function that checks for all the subfolders to add here.

Reply:
Now discovered dynamically, see setup.py